arch/common/text_section_offset_section.ld

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+SECTION_PROLOGUE(text_section_offset,,)
```

Comment:
There is text_section_offset.ld file in arch/common, do we need to have both @oyvindronningstad ?

Reply:
Yes, one is for ARM, the other is for RISCV and x86.

Reply:
OK, saw it. I think we need to implement this a bit better.

Reply:
Do you want more besides the comments you requested in the other comment?